components/compliance-service/reporting/relaxting/util_test.go

```diff
@@ -1,23 +1,30 @@
 package relaxting
 
 import (
-	"github.com/chef/automate/components/compliance-service/ingest/ingestic/mappings"
-	"github.com/stretchr/testify/assert"
 	"testing"
 	"time"
+
+	"github.com/chef/automate/components/compliance-service/ingest/ingestic/mappings"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestGetEsIndexNoTimeLast24h(t *testing.T) {
 	sum := mappings.IndexNameSum
 	filters := make(map[string][]string)
 
-	index, err := GetEsIndex(filters, true)
-
-	today := time.Now().UTC().Format("2006.01.02")
-	yesterday := time.Now().Add(-24 * time.Hour).UTC().Format("2006.01.02")
+	todayTime := time.Now().UTC()
 
-	assert.Equal(t, sum+"-"+yesterday+"*,"+sum+"-"+today+"*", index)
+	index, err := GetEsIndex(filters, true)
 	assert.NoError(t, err)
+
+	today := todayTime.Format("2006.01.02")
+	if today[len(today)-2:] == "31" {
+		//if it's the 31st, shave off the 1 and make it 3*, which will be the 30th and 31st
+		assert.Equal(t, sum+"-"+today[:len(today)-1]+"*", index)
```

Comment:
I t will always end up as `3*`. So better to mention that:
`assert.Equal(t, sum+"-3*", index)`

Reply:
done